### PR TITLE
create rebuild queue

### PR DESCRIPTION
**Pull request by zahidkizmaz**
# Descripton
This PR addresses the issue #2328 

Creates a new section under: releases/queue page called Rebuild queue. By default it only shows the count with an option to click "Show" would reveal the queue. 

##### Queue with 3 items expand unset:
20241022_08h19m51s_grim

##### Queue with 3 items and expand=1:
20241024_01h03m37s_grim

### CODE REVIEW

File: templates/style/style.scss [hunk 1]
```diff
@@ -7,18 +7,19 @@
     font-style: normal;
     font-weight: 400;
     src:
-      local('Fira Sans'),
-      url("FiraSans-Regular.woff2") format('woff2'),
-      url("FiraSans-Regular.woff") format('woff');
+        local('Fira Sans'),
```
Comment (zahidkizmaz) [inline templates/style/style.scss:10]: While adding new class, My editor ran `prettier` on this scss file, in order to make it `prettier`. Please let me know if changes are not fine I can just revert the prettier changes :)

File: src/web/releases.rs [hunk 2]
```diff
@@ -834,10 +843,14 @@ pub(crate) async fn build_queue_handler(
             })
         })
         .map(|mut krate| {
```
Comment (syphar) [inline src/web/releases.rs:845]: I think this should be `retain_mut` as in [my WIP branch](https://github.com/rust-lang/docs.rs/compare/master...syphar:docs.rs:group-build-queue?expand=1#diff-c3796aac6c6b01cd1998b29108b5d32fe5113970a7f4558e799c18984307d73aR819), or am I missing something?

File: templates/style/style.scss (hunk 1 above)
Comment (syphar) [reply templates/style/style.scss:10]: please revert these, so we can review only the SCSS changes. 

Any formatting should be separate and a separate discussion

**Review by syphar — CHANGES_REQUESTED** [review]
Thank you for this! 

Two comments, and could to reword the commit message of the initial `wip` commit? 

Generally we're doing rebase merges here

File: src/build_queue.rs [hunk 3]
```diff
@@ -15,6 +15,8 @@ use std::sync::Arc;
 use tokio::runtime::Runtime;
 use tracing::{debug, error, info};
 
+pub(crate) const REBUILD_PRIORITY: i32 = 20;
```
Comment (GuillaumeGomez) [inline src/build_queue.rs:18]: Might be worth adding a comment explaining what it does.

File: templates/releases/build_queue.html [hunk 4]
```diff
@@ -84,9 +84,40 @@
...
+                <strong>Rebuild Queue</strong>
+            </div>
+
+            <div class="about">
+                <p>
+                    We continuously rebuild the latest versions for all crates so they can
+                    benefit from new features in rustdoc.
+                </p>
+                {%- if !expand_rebuild_queue -%}
+                    <p>There are currently {{ rebuild_queue.len() }} crates in the rebuild queue.</p>
```
Comment (GuillaumeGomez) [inline templates/releases/build_queue.html:101]: Please use the [`pluralize`](https://rinja.readthedocs.io/en/stable/filters.html#pluralize) filter on `crates`.

**Comment by syphar** [comment]
@zahidkizmaz feel free to re-request reviews when you're ready

**Review by syphar — APPROVED** [review]

**Pull request merged**